docker

```diff
@@ -0,0 +1,5 @@
+e1b1ac128a42e5681143b853e02482b6804a57d6ec29d2e3ba14d0257db76744
```

Comment:
If this is a secret of any kind (e.g. a Trello token?) then we should rotate that so it's no longer valid, and clear it out from here